uint32 servo1
uint32 servo2
uint32 servo3
uint32 servo4
uint32 servo5
uint32 servo6